src/pages/viewer-authentication-api-v2-getting-started.mdx

```diff
@@ -33,18 +33,18 @@ If you embed the player to an unprotected page or use IBM's channel page, the pl
 You can enable viewer authentication via API.
 
 1. Generate an RSA key pair with 2048 bits key size
-1. Set the public key and authentication URL to the content (channel or video) via API. ([See more](/viewer-authentication-api-v2-settings-api#set-viewer-authentication-parameters-for-a-channel))
+1. Set the public key and authentication URL to the content (channel or video) via API. [See more](/viewer-authentication-api-v2-settings-api#set-viewer-authentication-parameters-for-a-channel)
     * and store the private key in your system secure way
 
     <InlineNotification kind="warning">
 
-    **DO NOT SHARE PRIVATE KEYS WITH US OR ANYBODY**
+    **Do not share private keys with us or anybody**
 
     </InlineNotification>
 
 We recommend using different key pairs for every content.
 
-By default, every video inherits the settings from the channel but if you want to use different authentication for a video you can set it. ([See more](/viewer-authentication-api-v2-settings-api#set-viewer-authentication-for-a-video))
+By default, every video inherits the settings from the channel but if you want to use different authentication for a video you can set it. [See more](/viewer-authentication-api-v2-settings-api#set-viewer-authentication-for-a-video)
```

Comment:
Suggestion:

>By default, every video inherits the settings from the channel...

By default, every video inherits these settings from the channel